<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Formmain</class>
 <widget class="QWidget" name="Formmain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>801</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Images/backkk.png);</string>
   </property>
   <widget class="QPushButton" name="btn_ParaSet">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>148</width>
      <height>148</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Images/softconfig.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Runstate">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>340</y>
      <width>211</width>
      <height>181</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Log">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>320</width>
      <height>148</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Images/history.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Inform">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>148</width>
      <height>148</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Images/state.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelTime">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>0000-00-00 00:00</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tablenowdate">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <zorder>btn_Runstate</zorder>
   <zorder>btn_ParaSet</zorder>
   <zorder>btn_Log</zorder>
   <zorder>btn_Inform</zorder>
   <zorder>labelTime</zorder>
   <zorder>tablenowdate</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
